tools/scripts/bumpenvs.py

```diff
@@ -11,7 +11,7 @@
 import sys
 from typing import List
 
-import yaml
+from determined.common import yaml
```

Comment:
actually, can you leave the standalone scripts alone?

I'd rather the standalone scripts not import determined just to import ruamel.yaml, that's like a huge amount of dependency bloat from the perspective of this script.